modules/core/src/main/java/org/apache/ignite/internal/processors/service/IgniteServiceProcessor.java

```diff
@@ -1061,23 +1061,6 @@ public ServiceContextImpl serviceContext(String name) {
     ) throws IgniteException {
         ctx.security().authorize(name, SecurityPermission.SERVICE_INVOKE);
 
-        if (hasLocalNode(prj)) {
```

Comment:
Now `IgniteServiceProcessor#hasLocalNode()` is obsolete and can be safely removed.